scripts/verify_clients_untracked_files.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# `--porcelain` gives the output in an easy-to-parse format for scripts
+git_status_output=$(git status clients/ --porcelain)
+
+untracked_files_found=false
+
+while IFS= read -r line; do
+    if [[ $line == \?\?* ]]; then
+        untracked_files_found=true
+	break
+    fi
+done <<< "$git_status_output"
+
+if $untracked_files_found; then
```

Comment:
This runs a command named in $untracked_files_found, which is somewhat scary.  If we're sticking with a solution based on shell variables, can we use a shell expression rather than hold the name of a program in the variable?  (But obviously I prefer just to grep and use the exit status directly in `if`, meaning we have fewer lines of sh and use it more as intended.)